opentelemetry-sdk/src/opentelemetry/sdk/util/instrumentation.py

```diff
@@ -132,12 +149,19 @@ def version(self) -> Optional[str]:
     def name(self) -> str:
         return self._name
 
+    @property
+    def attributes(self) -> Optional[Attributes]:
```

Comment:
```suggestion
    def attributes(self) -> Attributes:
```
shouldn't be None here i think since it is initialized to `BoundedAttributes` in `__init__`